runtime/job-workers/processing-jobs.cpp

```diff
@@ -3,36 +3,44 @@
 // Distributed under the GPL v3 License, see LICENSE.notice.txt
 
 #include "runtime/net_events.h"
-#include "runtime/job-workers/job-message.h"
-#include "runtime/job-workers/shared-memory-manager.h"
 #include "runtime/instance-copy-processor.h"
 
+#include "server/job-workers/job-message.h"
+#include "server/job-workers/shared-memory-manager.h"
+
 #include "runtime/job-workers/processing-jobs.h"
 
 namespace job_workers {
 
+struct FinishedJob {
```

Comment:
Can it be type alias?

Reply:
I believe, that the simple struct is a bit clearer. And also, I'm using the forward declaration for it, so the definition of `FinishedJob` can be a private.